language/Portuguese_Brazil/strings.po

```diff
@@ -1848,11 +1848,11 @@ msgstr "Informações"
 
 #: src/frontend/mame/ui/selmenu.cpp:2483
 msgid "Usage"
-msgstr ""
+msgstr "Uso"
 
 #: src/frontend/mame/ui/selmenu.cpp:2518
 msgid "General Info"
-msgstr "Información general"
+msgstr "Informação Geral"
```

Comment:
The more commonly used term in portuguese here would be plural: "Informações Gerais"